packages/labs/test/tagInputTests.tsx

```diff
@@ -4,7 +4,7 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import { assert } from "chai";
+import { assert, expect } from "chai";
```

Comment:
typically we try to use only one assertion style per file, but whatever. everyone prefers expect, but we have a lot of legacy asserts, so no big deal here.